### PR TITLE
Emit events from RateLimiter

### PR DESCRIPTION
**Pull request by lucasefe**
It adds MiniEE to RateLimiter to emit events with useful data. Also fixes some minor tests. 

### Steps to merge

:+1: of the team

@zendesk/zendesk-radar 

### Risks
none.

### CODE REVIEW

**Comment by bolddane** [comment]
:+1:

File: server/server.js [hunk 1]
```diff
@@ -297,7 +301,7 @@ Server.prototype._getRateLimiterForMessageType = function(messageType) {
 
     if (!rateLimiter) {
       // TODO: subscribe, as rate limiter operation, should be configurable. 
-      rateLimiter = new RateLimiter(messageType.policy.limit);
+      rateLimiter = this.buildRateLimiter(messageType);
```
Comment (vanchi-zendesk) [inline server/server.js]: I don't think this is actually needed. Let's not add more methods.

**Comment by vanchi-zendesk** [comment]
:+1:

File: server/server.js (hunk 1 above)
Comment (lucasefe) [inline server/server.js]: I just removed it in favor of emitting an event every time radar server
adds / builds a new rate limiter.

Lucas Florio
Software Engineer
Zendesk <http://www.zendesk.com>

**Comment by vanchi-zendesk** [comment]
:+1:

**Pull request merged**